test/test_parabolic_2d.jl

```diff
@@ -490,6 +490,24 @@ end
                         tspan=(0.0, 0.7))
 end
 
+@trixi_testset "TreeMesh2D: elixir_navierstokes_taylor_green_vortex_sutherland.jl" begin
+    @test_trixi_include(joinpath(examples_dir(), "tree_2d_dgsem",
+                                 "elixir_navierstokes_taylor_green_vortex_sutherland.jl"),
+                        l2=[
+                            0.001452856280034929,
+                            0.0007538775539989481,
+                            0.0007538775539988681,
+                            0.011035506549989587,
+                        ],
+                        linf=[
+                            0.003291912841311362,
+                            0.002986462478096974,
+                            0.0029864624780958637,
+                            0.0231954665514138,
+                        ],
+                        tspan=(0.0, 1.0))
```

Comment:
Just to confirm: This test runs in <10 seconds after compilation?

Reply:
Yes, actually less than a second on my computer.